ee/modules/140-user-authz/templates/webhook/configmap.yaml

```
@@ -8,9 +8,9 @@ metadata:
   {{- include "helm_lib_module_labels" (list . (dict "app" "user-authz-webhook")) | nindent 2 }}
 data:
   config.json: |
-    { "crds": {{ .Values.userAuthz.internal.crds | toJson}} }
+    { "clusterAuthRuleCrds": {{ .Values.userAuthz.internal.clusterAuthRuleCrds | toJson}} }
```

```suggestion
    { "crds": {{ .Values.userAuthz.internal.clusterAuthRuleCrds | toJson}} }
```
The name should still be `crds` there because the webhook controller dynamically reads this configmap, which is a race condition. Configuration may change while rolling the pod.

---

done